pymc3/backends/__init__.py

```diff
@@ -20,8 +20,12 @@
 in the directory 'test'.
 
     >>> import pymc3 as pm
-    >>> db = pm.backends.Text('test')
-    >>> trace = pm.sample(..., trace=db)
+    >>> with pm.Model():
+    >>>      db = pm.backends.Text('test')
+    >>>      trace = pm.sample(..., trace=db)
+
+Note that as in the example above, one must have an active model context,
+or pass a `model` parameter in order to create a backend.
```

Comment:
More fundamentally than that, you can only create a new backend in a call to `sample` (which is always called in a Model context)

Reply:
I don't actually understand your claim here.  The assignment to `db` in the code snippet above works, and is not in a call to `sample`.  So it seems to me that I am creating a backend outside the call to `sample`, and just *populating* it in the call to `sample`.  Is that not correct?  If not, what happens in line 24 above?  How should we describe that, if it's not creating a new backend?  Thanks.

Reply:
You are correct. I completely misread what you wrote. Apologies.